src/primal_dual_hybrid_gradient.jl

```diff
@@ -144,12 +144,18 @@ struct PdhgParameters
   pock_chambolle_alpha::Union{Float64,Nothing}
 
   """
-  Used to bias the computation of the primal/dual balancing parameter
-  primal_weight. Must be positive. A value of 1 balances primal and dual
-  equally.
+  Used to bias the initial value of the primal/dual balancing parameter
+  primal_weight. Must be positive. See scale_invariant_initial_primal_weight.
   """
   primal_importance::Float64
 
+  """
+  If true, computes the initial primal weight with a scale-invariant formula
+  biased by primal_importance; see select_initial_primal_weight. If false,
```

Comment:
(same comment as for mirror_prox.jl): Perhaps "see select_initial_primal_weight() for more details" to make it clearer that this is pointing into the code, and not at another field or somewhere else in the documentation.